timelock-server/src/main/java/com/palantir/atlasdb/timelock/clock/ClockServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.clock;
+
+public class ClockServiceImpl implements ClockService {
+    @Override
+    public long getSystemTimeInNanos() {
+        return System.nanoTime();
```

Comment:
@schlosna the goal here is to compare perceived differences in time among machines, not wall-clock time. i.e., we want to invoke `nanoTime` on machines A and B, at two points in time, and compare `t2_A - t1_A` to `t2_B - t1_B`